pycon-submission.md

```diff
@@ -12,87 +25,124 @@ Intermediate
+* An interactive, terminal-based shell with capabilities beyond the default
+  Python interactive interpreter (this is the classic application opened by the
+  `ipython` command that most users are familiar with).
 
-* A [web-based notebook](http://ipython.org/notebook.html) that can execute code and also contain rich text and figures, mathematical equations and arbitrary HTML. This notebook presents a document-like view with cells where code is executed but that can be edited in-place, reordered, mixed with explanatory text and figures, etc. The notebook provides an interactive experience that combines live code and results with literate documentation and the rich media that modern browsers can display:
+* A [web-based notebook](http://jupyter.org/) that can execute
+  code and also contain rich text and figures, mathematical equations and
+  arbitrary HTML. This notebook presents a document-like view with cells where
+  code is executed but that can be edited in-place, reordered, mixed with
+  explanatory text and figures, etc. The notebook provides an interactive
+  experience that combines live code and results with literate documentation
+  and the rich media that modern browsers can display:
 
-![Notebook screenshot](http://i.imgur.com/eo2SqS9.png)
+    ![Notebook screenshot](http://jupyter.org/assets/jupyterpreview.png)
 
-* A high-performance, low-latency system for [parallel computing](http://ipython.org/ipython-doc/stable/parallel/parallel_intro.html) that supports the control of a cluster of IPython engines communicating over ZeroMQ, with optimizations that minimize unnecessary copying of large objects (especially numpy arrays). These engines can be controlled interactively while developing and doing exploratory work, or can run in batch mode either on a local machine or in a large cluster/supercomputing environment via a batch scheduler.
+  The notebooks also allow for code in multiple languages allowing to mix Python
+  with Cython, C, R and other languages to access features hard to obain from
+  Python.
 
-These tools also increasingly work with languages other than Python, and we are renaming the language independent frontend components to *Jupyter* in order to make this clearer. The Python kernel we provide and the original terminal-based shell will continue to be called *IPython*.
+These tools also increasingly work with languages other than Python, and we
+renamed the language independent frontend components to *Jupyter* in order to
+make this clearer. The Python kernel we provide and the original terminal-based
+shell will continue to be called *IPython*.
 
 In this hands-on, in-depth tutorial, we will briefly describe IPython's
 architecture and will then show how to use the above tools for a highly
 productive workflow in Python.
 
 # Outline
 
```

Comment:
Note to reviewers: Each section will take 1/4 of the teaching time, taking into account the scheduled snack break. Each section will provide takeaway slides and notebooks for the attendee. There will be hands-on time of 5-10 minutes during each section for attendees to try out concepts.